### PR TITLE
String-Manipulation-1

### PR DESCRIPTION
**Pull request by Henry-Nichols**
image

### CODE REVIEW

File: string-manipulation-1/src/is-upper-cased.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+/* exported isUpperCased */
+function isUpperCased(word) {
+  if (word === word.toUpperCase()) {
+    return true;
+  } else {
+    return false;
+  }
```
Comment (codingcodymiller) [inline string-manipulation-1/src/is-upper-cased.js:7]: For this portion, you could just do this: `return word === word.toUpperCase()`

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Nice work so far! I have one improvement you could make.

File: string-manipulation-1/src/is-upper-cased.js (hunk 1 above)
Comment (Henry-Nichols) [reply string-manipulation-1/src/is-upper-cased.js]: Thanks Cody for the feedback!

File: string-manipulation-1/src/is-upper-cased.js [hunk 2]
```diff
@@ -1,8 +1,9 @@
...
-  if (word === word.toUpperCase()) {
-    return true;
-  } else {
-    return false;
-  }
+  // if (word === word.toUpperCase()) {
+  //   return true;
+  // } else {
+  //   return false;
+  // }
```
Comment (codingcodymiller) [inline string-manipulation-1/src/is-upper-cased.js:7]: Please remove commented code

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Review by codingcodymiller — APPROVED** [review]

**Pull request merged**